src/os/bluestore/KernelDevice.cc

```diff
@@ -142,6 +142,7 @@ int KernelDevice::open(const string& p)
     } else {
       dout(20) << __func__ << " devname " << devname << dendl;
       rotational = block_device_is_rotational(devname);
+      support_discard = block_device_support_discard(devname);
```

Comment:
Could you please print this value at the end of the function as we do with other params?

Reply:
@ifed01, yep

Reply:
Done